packages/api/src/controllers/stream.ts

```diff
@@ -293,65 +294,100 @@ export function getFLVPlaybackUrl(ingest: string, stream: DBStream) {
  * Returns whether the stream is currently tagged as active but hasn't been
  * updated in a long time and thus should be cleaned up.
  */
-function shouldActiveCleanup(stream: DBStream | DBSession) {
-  const isActive = "isActive" in stream ? stream.isActive : true; // sessions don't have `isActive` field so we just assume `true`
+function shouldCleanUpIsActive(stream: DBStream | DBSession) {
+  // sessions don't have `isActive` field so we just assume `true`
+  const isActive =
+    stream.kind === "stream" && "isActive" in stream ? stream.isActive : true;
   return (
     isActive &&
     !isNaN(stream.lastSeen) &&
     Date.now() - stream.lastSeen > ACTIVE_TIMEOUT
   );
 }
 
-function activeCleanupOne(
+/**
+ * Creates an updated stream object with a fixed `isActive` field in case it is lost and needs clean up.
+ */
+function withFixedIsActive(stream: DBStream) {
```

Comment:
Does it mean that we can have a stream object without `isActive` field? 

I wonder if we should fix such streams in the clean-up logic. I think the sematic of not having `isActive` field would be that the stream is not active. So, probably no need to clean it up.

Reply:
So this was just to keep the same logic of returning `isActive: false` after the active timeout for streams that got lost in action. Similar to what had before, but instead of triggering the clean-up job at that point we just monkey patch the API object before returning it, and wait for the clean-up job to pick it up.

I made it like this to make sure we keep retrocompatibility, tho thinking further about it I thought that we could also just not have the monkey patch at all? So we keep returning `isActive: true` for the user for up to ~7 minutes (worst case) until a clean-up job comes and fixes it (and triggers recording).

WDYT? Should we keep the retrocompatibility in the sense "if the stream is not seen in 90s we already start returning `isActive: false`", or should we just remove this monkey patch once and for all?